workflows/hobgoblin-reactiontime-samplediscreteuniform.bonsai

```diff
@@ -18,7 +18,7 @@
           <p1:Upper>1</p1:Upper>
         </Combinator>
       </Expression>
-      <Expression xsi:type="rx:PublishSubject">
+      <Expression xsi:type="rx:BehaviorSubject">
```

Comment:
@banchan86 nice catch! Actually the best subject to use here would be `AsyncSubject` since we want to keep the distribution itself immutable (read-only variable, can be sampled multiple times of course).